### DISCUSSION TITLE
Using callouts.

### DISCUSSION THREAD

**Discussion by awalsh128** [mmistakes/minimal-mistakes#2709] [Q&A] [answered]
The default doesn't include the templates for callouts in an includes directory. I copied the includes over.

```
<div markdown="span" class="alert alert-warning" role="alert" ><i class="fa fa-warning"></i> <b>Important:</b> {{include.content}}</div>
```

Although, when I use them I don't get the callout background, just the icon.

```
{% include note.html 
    content="This is my note. All the content I type here is 
    treated as a single paragraph." %}
```

What can I do to get the colored callouts? Is there something I am missing or is it just not available?

**Comment by mmistakes** [comment, marked as answer]
Here’s how I do them in the theme’s documentation:

https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_posts/2010-02-05-post-notice.md

**Comment by awalsh128** [comment]
Perfect, thanks. I ended up modifying these in the includes. Something like:

`<p><i class="fa fa-exclamation"></i> <b>IMPORTANT:</b> {{include.content}}</p>{: .notice--danger}`
